libsql-server/src/lib.rs

```diff
@@ -456,6 +456,7 @@ where
                 namespace_store_shutdown_fut.await?;
                 // clean shutdown, remove sentinel file
                 std::fs::remove_file(sentinel_file_path(&self.path))?;
+                println!("server shutdown complete -- clean exit");
```

Comment:
please use `tracing::info!` for that

Reply:
hey yes, I have reverted it to `trace`. I wanted to check if the tests time out, so I pushed something quickly. 

@LucioFranco I think the timeout setting is not working as expected. There is a deadlock in my PR and this particular `test::bottomless::rollback_restore` test never finishes. However, its been more than 10 mins and CI hasn't cancelled it.